src/main/java/de/nplay/moderationbot/permissions/BotPermissionsCommands.java

```diff
@@ -0,0 +1,131 @@
+    public void onPermissionsList(CommandEvent event, @Optional Member member) {
+        Member target = member == null ? event.getMember() : member;
+
+        assert target != null;
+        event.reply(embedCache.getEmbed("permissionsList")
+                .injectValue("target", target.getEffectiveName())
+                .injectValue("permissions", BotPermissions.getPermissionListString(BotPermissionsService.getMemberPermissions(target).permissions()))
+                .injectValue("color", EmbedColors.DEFAULT.hexColor)
+        );
+    }
+
+    @SlashCommand(value = "permissions manage member", desc = "Verwaltet die Berechtigungen eines Benutzers.", isGuildOnly = true, enabledFor = Permission.BAN_MEMBERS)
+    @Permissions(BotPermissionFlags.PERMISSION_MANAGE)
+    public void onManageMemberPermissions(CommandEvent event, Member member) {
+        targetMember = member;
+
+        var menu = event.getSelectMenu(
+                        "BotPermissionsCommands.onMemberPermissionSelect",
+                        net.dv8tion.jda.api.interactions.components.selections.StringSelectMenu.class)
+                .createCopy();
+        menu.getOptions().clear();
+        menu.setMaxValues(SelectMenu.OPTIONS_MAX_AMOUNT);
+
+        for (BotPermissionBitfield permission : BotPermissionBitfield.values()) {
+            menu.addOption(permission.humanReadableName, permission.name());
+        }
+
+        menu.addOption("Keine Berechtigungen (löscht automatisch alle)", "NONE");
+
+        var currentPermissions = BotPermissionsService.getMemberPermissions(member).permissions();
+        menu.setDefaultValues(BotPermissions.decodePermissions(currentPermissions).stream().map(Enum::name).toList());
+
+        event.getReplyContext().getBuilder().addActionRow(menu.build());
+        event.reply(
+                embedCache.getEmbed("permissionsEdit")
+                        .injectValue("target", member.getEffectiveName())
+                        .injectValue("color", EmbedColors.DEFAULT.hexColor)
+        );
+    }
+
+    @SlashCommand(value = "permissions manage role", desc = "Verwaltet die Berechtigungen einer Rolle.", isGuildOnly = true, enabledFor = Permission.BAN_MEMBERS)
+    @Permissions(BotPermissionFlags.PERMISSION_MANAGE)
+    public void onManageRolePermissions(CommandEvent event, Role role) {
+        targetRole = role;
+
+        var menu = event.getSelectMenu(
+                        "BotPermissionsCommands.onRolePermissionSelect",
+                        net.dv8tion.jda.api.interactions.components.selections.StringSelectMenu.class)
+                .createCopy();
+        menu.getOptions().clear();
+        menu.setMaxValues(SelectMenu.OPTIONS_MAX_AMOUNT);
+
+        for (BotPermissionBitfield permission : BotPermissionBitfield.values()) {
+            menu.addOption(permission.humanReadableName, permission.name());
+        }
+
+        menu.addOption("Keine Berechtigungen (löscht automatisch alle)", "NONE");
```

Comment:
since we use this option more than once we could define it as a static class field